data/routing.txt

```diff
@@ -15,8 +14,15 @@ Bitcoin         | Coinbase   | 1% + 15¢     | On request |
 
 Route           | Provider   | Fee          | When       | Details
 ----------------+------------+--------------+------------+-----------
-Bank Account    | Balanced   | 0            | At payday  | U.S. only
```

Comment:
We're still on Balanced for ACH for this week. We are aiming to be on Zipmark for next week.